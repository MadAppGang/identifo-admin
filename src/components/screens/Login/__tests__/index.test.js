import LoginScreen from '..';

describe('<LoginScreen />', () => {
  test('renders as expected', () => {
    expect(shallow(<LoginScreen />)).toMatchSnapshot();
  });
});
