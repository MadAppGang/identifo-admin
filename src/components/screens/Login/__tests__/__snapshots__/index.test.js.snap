// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoginScreen /> renders as expected 1`] = `
<LoginScreenLayout>
  <Connect(LoginForm) />
</LoginScreenLayout>
`;
